sos/report/plugins/infinidat.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2024 Alejandro Santoyo <alejandro.santoyo@canonical.com>
+#
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+
+import platform
+from sos.report.plugins import Plugin, IndependentPlugin
+    
+    
+class InfinidatStoragePlugin(Plugin, IndependentPlugin):
```

Comment:
Pedantic nit: we don't generally suffix our plugin class names with 'Plugin', so this would usually just be `InfinidatStorage` or simply `Infinidat`.

Reply:
Fixed, thanks